libs/vertexai/langchain_google_vertexai/ranker/rank.py

```diff
@@ -0,0 +1,155 @@
+from typing import Any, List, Optional
+
+from google.api_core import exceptions as core_exceptions
+from google.auth.credentials import Credentials
+from google.cloud import discoveryengine_v1alpha
+from langchain.retrievers.document_compressors.base import BaseDocumentCompressor
+from langchain_core.documents import Document
+from langchain_core.pydantic_v1 import BaseModel, Extra, Field
+
+from langchain_google_vertexai.ranker._sdk_manager import VertexRankSDKManager
+
+
+class VertexAIRanker(BaseDocumentCompressor, BaseModel):
```

Comment:
can we inherit from `BaseDocumentCompressor` and comply to its interface, please?